assets/apps.json

```diff
@@ -1,11 +1,7 @@
 {
     "apps" : {
-        "jira" : "v3.0.0",
-        "todo" : "v2.2.3"
+        "zendesk" : "0.0.1"
```

Comment:
I've update the file to only contain "valid" data as zendesk is the only app that runs in AWS right now.